File: LeetCode/1431. Kids With the Greatest Number of Candies.py
class Solution(object):
    def kidsWithCandies(self, candies, extraCandies):
        """
        :type candies: List[int]
        :type extraCandies: int
        :rtype: List[bool]
        """
        mx = max(candies)
        ans = [False] * len(candies)
        for i in range(len(candies)):
            if candies[i]+extraCandies >= mx:
                 ans[i] = True
        return ans
        

File: LeetCode/1431. Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int mx = *max_element(candies.begin(), candies.end());
        int n = candies.size();
        vector<bool> ans(n,0);
        for(int i = 0;  i <n; i++){
            if(candies[i]+extraCandies >= mx) ans[i] = 1;
        }
        return ans;
    }
};